Source/Fuse.Navigation/TriggerActions.uno

```diff
@@ -115,10 +112,7 @@ namespace Fuse.Navigation
 
 		protected override void Perform(INavigation ctx, Node n)
 		{
-			var v = n as Visual;
-			if (v == null) return;
-
-			var page = Navigation.TryFindPage(Target ?? v);
+			var page = Navigation.TryFindPage((Node)Target ?? n);
```

Comment:
I don'tthink you need the cast here. `Target` is a `Visual`, and a `Visual` is a `Node`...